py_ecc/optimized_bn128/optimized_curve.py

```diff
@@ -142,15 +142,13 @@ def normalize(pt: Optimized_Point3D[Optimized_Field]) -> Optimized_Point2D[Optim
 
 # Convert P => -P
 def neg(pt: Optimized_Point3D[Optimized_Field]) -> Optimized_Point3D[Optimized_Field]:
-    if pt is None:
-        return None
+    if is_inf(pt):
+        return pt
     x, y, z = pt
     return (x, -y, z)
 
 
 def twist(pt: Optimized_Point3D[FQP]) -> Optimized_Point3D[FQP]:
-    if pt is None:
-        return None
```

Comment:
This removal of None check makes sense to me. Since points at infinity Z1 and Z2 are tuples in this case, None check is pointless.